### ISSUE TITLE
zero new files found on path

### ISSUE THREAD

**Issue by bmgeek** [fluent/fluent-bit#2382]
##[input:tail:tail.0] 0 new files found on path '/var/log/containers/*.log'

I set up logging of kubernetes with output to kafka. I see this in debug:

```
[2020/07/22 17:02:21] [ info] Configuration:
[2020/07/22 17:02:21] [ info]  flush time     | 1.000000 seconds
[2020/07/22 17:02:21] [ info]  grace          | 5 seconds
[2020/07/22 17:02:21] [ info]  daemon         | 0
[2020/07/22 17:02:21] [ info] ___________
[2020/07/22 17:02:21] [ info]  inputs:
[2020/07/22 17:02:21] [ info]      tail
[2020/07/22 17:02:21] [ info] ___________
[2020/07/22 17:02:21] [ info]  filters:
[2020/07/22 17:02:21] [ info]      kubernetes.0
[2020/07/22 17:02:21] [ info] ___________
[2020/07/22 17:02:21] [ info]  outputs:
[2020/07/22 17:02:21] [ info]      kafka.0
[2020/07/22 17:02:21] [ info] ___________
[2020/07/22 17:02:21] [ info]  collectors:
[2020/07/22 17:02:21] [ info] [engine] started (pid=1)
[2020/07/22 17:02:21] [debug] [engine] coroutine stack size: 24576 bytes (24.0K)
[2020/07/22 17:02:21] [debug] [storage] [cio stream] new stream registered: tail.0
[2020/07/22 17:02:21] [ info] [storage] version=1.0.4, initializing...
[2020/07/22 17:02:21] [ info] [storage] in-memory
[2020/07/22 17:02:21] [ info] [storage] normal synchronization mode, checksum disabled, max_chunks_up=128
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] inotify watch fd=20
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] scanning path /var/log/containers/*.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/calico-node-tsft7_kube-system_calico-node-324fafed5cf300cbeb621276d15f60ade06cf9b2b3f93c9e48e993967f7e7fcf.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/calico-node-tsft7_kube-system_calico-node-8fa067ceade921d39bcebc83f46b7f541ddba6818d34b6eeb70542de24d8ad5f.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/calico-node-tsft7_kube-system_flexvol-driver-0eb84ff65b743f39092504acce76316f886f9b1bb3404c00b68a9737b362fa18.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/calico-node-tsft7_kube-system_install-cni-e729bb3558991717164e1b7495c3aed77b37705389d7f15723395af402a90e66.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/calico-node-tsft7_kube-system_upgrade-ipam-8bcf447393dc54b7fd394ca1905569c674dedc7ad4cf93902ca6afdc40161111.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/etcd-k8s-master01d_kube-system_etcd-01c780ff9a75b98a7a9ea5495780492ebb3bd4924f7d50fda27c50f725c8a282.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/fluent-bit-zsz7n_kube-logging_fluent-bit-fb646019b5218914cc1c5bc9652fc97e2c0af7dcf014a0e9e9e55c0a400c70cf.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-apiserver-k8s-master01d_kube-system_kube-apiserver-7e926100ba088ef125d88b18ec8880f27c07e2becc051345c3537094a35df548.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-apiserver-k8s-master01d_kube-system_kube-apiserver-81f31525c81762cf4c5c7592d655407816990e87e50793140cc1b3b66b3495a3.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-controller-manager-k8s-master01d_kube-system_kube-controller-manager-8573651e44178ae0d77aa77b93d94c272c9489af3d4c6bd211ba2aa77fbf1a8b.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-controller-manager-k8s-master01d_kube-system_kube-controller-manager-95df9d931cd60b3703ebabd250f25faec346e503d91c7ae09f6e9236366064cf.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-proxy-q4jvz_kube-system_kube-proxy-aa7bce0589897b7a38b4c1b4f154c72c241522b0253d2e4eb2066545dc8cd161.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-proxy-q4jvz_kube-system_kube-proxy-ae482b1dd96ff9ec6afca53cff37b3dba0312a87fd14970a437cb86dce322d0b.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-scheduler-k8s-master01d_kube-system_kube-scheduler-38b8293093e914aa1152280b589552e54f518fe5a2d3e2dc809ed83afd074d38.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-scheduler-k8s-master01d_kube-system_kube-scheduler-3fba3e03f62efb5e2c813a515cad4909a4c1695df909648cc425883cb7ded9f1.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/speaker-nmc2x_metallb-system_speaker-9f88d20f4d9c65c920b1c57c2a1bb60c1fa66cffbce6e7739caa58f4c02981b4.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/speaker-nmc2x_metallb-system_speaker-b9e707e2e2196ac3b39f8486c6eeba0d53c943e6bc8c896101bee5ae3ac54199.log
[2020/07/22 17:02:21] [debug] [input:tail:tail.0] 0 new files found on path '/var/log/containers/*.log'
[2020/07/22 17:02:21] [ info] [filter:kubernetes:kubernetes.0] https=1 host=kubernetes.default.svc port=443
[2020/07/22 17:02:21] [ info] [filter:kubernetes:kubernetes.0] local POD info OK
[2020/07/22 17:02:21] [ info] [filter:kubernetes:kubernetes.0] testing connectivity with API server...
[2020/07/22 17:02:21] [debug] [http_client] server kubernetes.default.svc:443 will close connection #21
[2020/07/22 17:02:21] [debug] [filter:kubernetes:kubernetes.0] API Server (ns=kube-logging, pod=fluent-bit-zsz7n) http_do=0, HTTP Status: 200
[2020/07/22 17:02:21] [ info] [filter:kubernetes:kubernetes.0] API server connectivity OK
[2020/07/22 17:02:21] [ info] [output:kafka:kafka.0] brokers='logging-elastic-bal:9092' topics='k8s-logs'
[2020/07/22 17:02:21] [debug] [router] match rule tail.0:kafka.0
[2020/07/22 17:02:21] [ info] [http_server] listen iface=0.0.0.0 tcp_port=2020
[2020/07/22 17:02:21] [ info] [sp] stream processor started
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] scanning path /var/log/containers/*.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/calico-node-tsft7_kube-system_calico-node-324fafed5cf300cbeb621276d15f60ade06cf9b2b3f93c9e48e993967f7e7fcf.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/calico-node-tsft7_kube-system_calico-node-8fa067ceade921d39bcebc83f46b7f541ddba6818d34b6eeb70542de24d8ad5f.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/calico-node-tsft7_kube-system_flexvol-driver-0eb84ff65b743f39092504acce76316f886f9b1bb3404c00b68a9737b362fa18.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/calico-node-tsft7_kube-system_install-cni-e729bb3558991717164e1b7495c3aed77b37705389d7f15723395af402a90e66.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/calico-node-tsft7_kube-system_upgrade-ipam-8bcf447393dc54b7fd394ca1905569c674dedc7ad4cf93902ca6afdc40161111.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/etcd-k8s-master01d_kube-system_etcd-01c780ff9a75b98a7a9ea5495780492ebb3bd4924f7d50fda27c50f725c8a282.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/fluent-bit-zsz7n_kube-logging_fluent-bit-fb646019b5218914cc1c5bc9652fc97e2c0af7dcf014a0e9e9e55c0a400c70cf.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-apiserver-k8s-master01d_kube-system_kube-apiserver-7e926100ba088ef125d88b18ec8880f27c07e2becc051345c3537094a35df548.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-apiserver-k8s-master01d_kube-system_kube-apiserver-81f31525c81762cf4c5c7592d655407816990e87e50793140cc1b3b66b3495a3.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-controller-manager-k8s-master01d_kube-system_kube-controller-manager-8573651e44178ae0d77aa77b93d94c272c9489af3d4c6bd211ba2aa77fbf1a8b.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-controller-manager-k8s-master01d_kube-system_kube-controller-manager-95df9d931cd60b3703ebabd250f25faec346e503d91c7ae09f6e9236366064cf.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-proxy-q4jvz_kube-system_kube-proxy-aa7bce0589897b7a38b4c1b4f154c72c241522b0253d2e4eb2066545dc8cd161.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-proxy-q4jvz_kube-system_kube-proxy-ae482b1dd96ff9ec6afca53cff37b3dba0312a87fd14970a437cb86dce322d0b.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-scheduler-k8s-master01d_kube-system_kube-scheduler-38b8293093e914aa1152280b589552e54f518fe5a2d3e2dc809ed83afd074d38.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/kube-scheduler-k8s-master01d_kube-system_kube-scheduler-3fba3e03f62efb5e2c813a515cad4909a4c1695df909648cc425883cb7ded9f1.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/speaker-nmc2x_metallb-system_speaker-9f88d20f4d9c65c920b1c57c2a1bb60c1fa66cffbce6e7739caa58f4c02981b4.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] skip (invalid) entry=/var/log/containers/speaker-nmc2x_metallb-system_speaker-b9e707e2e2196ac3b39f8486c6eeba0d53c943e6bc8c896101bee5ae3ac54199.log
[2020/07/22 17:02:26] [debug] [input:tail:tail.0] 0 new files found on path '/var/log/containers/*.log'
```

##My config

```
kind: ConfigMap
apiVersion: v1
metadata:
  name: fluentd-kafka-config
  namespace: kube-logging
  labels:
    addonmanager.kubernetes.io/mode: Reconcile
data:
  # Configuration files: server, input, filters and output
  # ======================================================
  fluent-bit.conf: |
    [SERVICE]
        Flush         1
        Log_Level     trace
        Daemon        off
        Parsers_File  parsers.conf
        HTTP_Server   On
        HTTP_Listen   0.0.0.0
        HTTP_Port     2020
    @INCLUDE input-kubernetes.conf
    @INCLUDE filter-kubernetes.conf
    @INCLUDE output-kafka.conf
  input-kubernetes.conf: |
    [INPUT]
        Name              tail
        Tag               kube.<namespace_name>.<pod_name>.<container_name>
        Tag_Regex         (?<pod_name>[a-z0-9](?:[-a-z0-9]*[a-z0-9])?(?:\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*)_(?<namespace_name>[^_]+)_(?<container_name>.+)-(?<docker_id>[a-z0-9]{64})\.log$
        Path              /var/log/containers/*.log
        Parser            docker
        DB                /var/flb_kube.db
        DB.Sync           Normal
        Mem_Buf_Limit     512MB
        Skip_Long_Lines   On
        Refresh_Interval  5
  filter-kubernetes.conf: |
    [FILTER]
        Name                kubernetes
        Match               kube.*
        Regex_Parser        k8s-custom-tag
        Kube_Tag_Prefix     kube.*
        Keep_Log            Off
        Merge_Log           On
        K8S-Logging.Parser  Off
        Annotations         Off
        K8S-Logging.Exclude Off
  output-kafka.conf: |
    [OUTPUT]
        Name           kafka
        Match          kube.*
        Brokers        logging-elastic-bal:9092
        Topics         k8s-logs
        #Timestamp_Key  @timestamp
        Retry_Limit    false
        # hides errors "Receive failed: Disconnected" when kafka kills idle connections
        rdkafka.log.connection.close false
        # producer buffer is not included in http://fluentbit.io/documentation/0.12/configuration/memory_usage.html#estimating
        rdkafka.queue.buffering.max.kbytes 10240
        # for logs you'll probably want this ot be 0 or 1, not more
        rdkafka.request.required.acks 1
  parsers.conf: |
    [PARSER]
        Name   apache
        Format regex
        Regex  ^(?<host>[^ ]*) [^ ]* (?<user>[^ ]*) \[(?<time>[^\]]*)\] "(?<method>\S+)(?: +(?<path>[^\"]*?)(?: +\S*)?)?" (?<code>[^ ]*) (?<size>[^ ]*)(?: "(?<referer>[^\"]*)" "(?<agent>[^\"]*)")?$
        Time_Key time
        Time_Format %d/%b/%Y:%H:%M:%S %z
    [PARSER]
        Name   apache2
        Format regex
        Regex  ^(?<host>[^ ]*) [^ ]* (?<user>[^ ]*) \[(?<time>[^\]]*)\] "(?<method>\S+)(?: +(?<path>[^ ]*) +\S*)?" (?<code>[^ ]*) (?<size>[^ ]*)(?: "(?<referer>[^\"]*)" "(?<agent>[^\"]*)")?$
        Time_Key time
        Time_Format %d/%b/%Y:%H:%M:%S %z
    [PARSER]
        Name   apache_error
        Format regex
        Regex  ^\[[^ ]* (?<time>[^\]]*)\] \[(?<level>[^\]]*)\](?: \[pid (?<pid>[^\]]*)\])?( \[client (?<client>[^\]]*)\])? (?<message>.*)$
    [PARSER]
        Name   nginx
        Format regex
        Regex ^(?<remote>[^ ]*) (?<host>[^ ]*) (?<user>[^ ]*) \[(?<time>[^\]]*)\] "(?<method>\S+)(?: +(?<path>[^\"]*?)(?: +\S*)?)?" (?<code>[^ ]*) (?<size>[^ ]*)(?: "(?<referer>[^\"]*)" "(?<agent>[^\"]*)")?$
        Time_Key time
        Time_Format %d/%b/%Y:%H:%M:%S %z
    [PARSER]
        Name   json
        Format json
        Time_Key time
        Time_Format %d/%b/%Y:%H:%M:%S %z
    [PARSER]
        Name        docker
        Format      json
        Time_Key    time
        Time_Format %Y-%m-%dT%H:%M:%S.%L
        Time_Keep   On
    [PARSER]
        Name        syslog
        Format      regex
        Regex       ^\<(?<pri>[0-9]+)\>(?<time>[^ ]* {1,2}[^ ]* [^ ]*) (?<host>[^ ]*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?(?:[^\:]*\:)? *(?<message>.*)$
        Time_Key    time
        Time_Format %b %d %H:%M:%S
    [PARSER]
        Name    k8s-custom-tag
        Format  regex
        Regex   ^(?<namespace_name>[^.]+)\.(?<pod_name>[^.]+)\.(?<container_name>[^.]+)$
```

**Comment by edsiper** [comment]
If you use the ```fluent/fluent-bit:1.5-debug```` image and get inside the container, can you list the content of /var/log/containers/* ?, I suspect this is like a setup issue when mounting the volumes

**Comment by bmgeek** [comment]
> If you use the `fluent/fluent-bit:1.5-debug` image and get inside the container, can you list the content of /var/log/containers/* ?, I suspect this is like a setup issue when mounting the volumes

Thanks for the answer. i will try and write the result

**Comment by bmgeek** [comment]
yes, it is symlink for another path =(
then how should i do?

use path /var/lib/docker/containers/*/*.log ?

```
drwxr-xr-x    2 root     root          4096 Jul 23 07:17 .
drwxr-xr-x    1 root     root          4096 Jul 23 07:17 ..
lrwxrwxrwx    1 root     root            98 Jun 18 09:46 calico-node-tsft7_kube-system_calico-node-324fafed5cf300cbeb621276d15f60ade06cf9b2b3f93c9e48e993967f7e7fcf.log -> /var/log/pods/kube-system_calico-node-tsft7_d9db517a-7544-40ff-9739-0e6fa6703313/calico-node/9.log
lrwxrwxrwx    1 root     root            98 Jun 17 21:50 calico-node-tsft7_kube-system_calico-node-8fa067ceade921d39bcebc83f46b7f541ddba6818d34b6eeb70542de24d8ad5f.log -> /var/log/pods/kube-system_calico-node-tsft7_d9db517a-7544-40ff-9739-0e6fa6703313/calico-node/8.log
lrwxrwxrwx    1 root     root           101 Jun 18 09:46 calico-node-tsft7_kube-system_flexvol-driver-0eb84ff65b743f39092504acce76316f886f9b1bb3404c00b68a9737b362fa18.log -> /var/log/pods/kube-system_calico-node-tsft7_d9db517a-7544-40ff-9739-0e6fa6703313/flexvol-driver/0.log
lrwxrwxrwx    1 root     root            98 Jun 18 09:46 calico-node-tsft7_kube-system_install-cni-e729bb3558991717164e1b7495c3aed77b37705389d7f15723395af402a90e66.log -> /var/log/pods/kube-system_calico-node-tsft7_d9db517a-7544-40ff-9739-0e6fa6703313/install-cni/0.log
lrwxrwxrwx    1 root     root           100 Jun 18 09:46 calico-node-tsft7_kube-system_upgrade-ipam-8bcf447393dc54b7fd394ca1905569c674dedc7ad4cf93902ca6afdc40161111.log -> /var/log/pods/kube-system_calico-node-tsft7_d9db517a-7544-40ff-9739-0e6fa6703313/upgrade-ipam/10.log
lrwxrwxrwx    1 root     root            90 Jul  5 06:35 etcd-k8s-master01d_kube-system_etcd-01c780ff9a75b98a7a9ea5495780492ebb3bd4924f7d50fda27c50f725c8a282.log -> /var/log/pods/kube-system_etcd-k8s-master01d_31e20009e11fc8901ae8c29a32b9df66/etcd/205.log
lrwxrwxrwx    1 root     root            97 Jul 23 07:17 fluent-bit-8j49p_kube-logging_fluent-bit-5ff2df19b857615d883d0f909aec44a9c999f09994fd3467f9def137938f2764.log -> /var/log/pods/kube-logging_fluent-bit-8j49p_904ea5af-1157-4203-8267-db3eaa88d05e/fluent-bit/0.log
lrwxrwxrwx    1 root     root           109 Jun 18 09:56 kube-apiserver-k8s-master01d_kube-system_kube-apiserver-7e926100ba088ef125d88b18ec8880f27c07e2becc051345c3537094a35df548.log -> /var/log/pods/kube-system_kube-apiserver-k8s-master01d_9cafec68a0e9bfb50a79b435ea3b13c0/kube-apiserver/75.log
lrwxrwxrwx    1 root     root           109 Jun 18 09:46 kube-apiserver-k8s-master01d_kube-system_kube-apiserver-81f31525c81762cf4c5c7592d655407816990e87e50793140cc1b3b66b3495a3.log -> /var/log/pods/kube-system_kube-apiserver-k8s-master01d_9cafec68a0e9bfb50a79b435ea3b13c0/kube-apiserver/76.log
lrwxrwxrwx    1 root     root           128 Jul 20 13:58 kube-controller-manager-k8s-master01d_kube-system_kube-controller-manager-8573651e44178ae0d77aa77b93d94c272c9489af3d4c6bd211ba2aa77fbf1a8b.log -> /var/log/pods/kube-system_kube-controller-manager-k8s-master01d_b4f2748568d79214c7f00ddecbd6f7b0/kube-controller-manager/737.log
lrwxrwxrwx    1 root     root           128 Jul 21 10:08 kube-controller-manager-k8s-master01d_kube-system_kube-controller-manager-95df9d931cd60b3703ebabd250f25faec346e503d91c7ae09f6e9236366064cf.log -> /var/log/pods/kube-system_kube-controller-manager-k8s-master01d_b4f2748568d79214c7f00ddecbd6f7b0/kube-controller-manager/738.log
lrwxrwxrwx    1 root     root            96 Jun 18 09:45 kube-proxy-q4jvz_kube-system_kube-proxy-aa7bce0589897b7a38b4c1b4f154c72c241522b0253d2e4eb2066545dc8cd161.log -> /var/log/pods/kube-system_kube-proxy-q4jvz_5d0a5fe4-a25b-4cc9-b232-7d18ff454275/kube-proxy/9.log
lrwxrwxrwx    1 root     root            96 Jun 17 21:49 kube-proxy-q4jvz_kube-system_kube-proxy-ae482b1dd96ff9ec6afca53cff37b3dba0312a87fd14970a437cb86dce322d0b.log -> /var/log/pods/kube-system_kube-proxy-q4jvz_5d0a5fe4-a25b-4cc9-b232-7d18ff454275/kube-proxy/8.log
lrwxrwxrwx    1 root     root           110 Jul 21 15:34 kube-scheduler-k8s-master01d_kube-system_kube-scheduler-38b8293093e914aa1152280b589552e54f518fe5a2d3e2dc809ed83afd074d38.log -> /var/log/pods/kube-system_kube-scheduler-k8s-master01d_13ec857b09eb10942d3c3c79d10b7afe/kube-scheduler/692.log
lrwxrwxrwx    1 root     root           110 Jul 21 06:12 kube-scheduler-k8s-master01d_kube-system_kube-scheduler-3fba3e03f62efb5e2c813a515cad4909a4c1695df909648cc425883cb7ded9f1.log -> /var/log/pods/kube-system_kube-scheduler-k8s-master01d_13ec857b09eb10942d3c3c79d10b7afe/kube-scheduler/691.log
lrwxrwxrwx    1 root     root            94 Jun 18 09:46 speaker-nmc2x_metallb-system_speaker-9f88d20f4d9c65c920b1c57c2a1bb60c1fa66cffbce6e7739caa58f4c02981b4.log -> /var/log/pods/metallb-system_speaker-nmc2x_9c859a39-76b2-4e54-9fd8-be53c8961c9a/speaker/10.log
lrwxrwxrwx    1 root     root            93 Jun 18 09:45 speaker-nmc2x_metallb-system_speaker-b9e707e2e2196ac3b39f8486c6eeba0d53c943e6bc8c896101bee5ae3ac54199.log -> /var/log/pods/metallb-system_speaker-nmc2x_9c859a39-76b2-4e54-9fd8-be53c8961c9a/speaker/9.log
```

**Comment by bmgeek** [comment]
i'm stupid!

it was not necessary not to remove the folder mounting from the standard config: /var/log/ and /var/lib/docker/containers

a lot of thanks for help  =)

**Comment by xuchenCN** [comment]
@bmgeek your scan folder is /var/lib/docker/containers? then why use the kubernetes filter?

**Comment by cobb-tx** [comment]
@bmgeek  how to fix this problem？

**Comment by cobb-tx** [comment]
> @bmgeek how to fix this problem？

this is ok.not problem.

**Comment by xuchenCN** [comment]
@cobb-tx @bmgeek check this [#2676]

**Comment by malekerramli** [comment]
whats the fix here ? im having the same issue

**Comment by tbernacchi** [comment]
I'd to know too =/

**Comment by eagle4567** [comment]
I tried the new fluent-bit image but still encounter same issue. Checked that there is no symlink to the log files too.

Have attached the updated yaml file and error result.

Files in /app/logs/splunk folder
2021-11-22-0-test-xxx-xxx-debug.log
2021-11-22-0-test-xxx-xxx-error.log
test-xxx-xxx-debug.log
test-xxx-xxx-error.log
test-xxx-xxx-xxxx.2021-11-22-0.l
[Error after fluent-bit 1.8.10.txt](https://github.com/fluent/fluent-bit/files/7599884/Error.after.fluent-bit.1.8.10.txt)
o
[fluentbit-dev-v3-2 template.yaml.txt](https://github.com/fluent/fluent-bit/files/7599885/fluentbit-dev-v3-2.template.yaml.txt)
g
